### PR TITLE
adds KMS quickstart

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: kms/quickstart/phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2016 Google Inc.
```
Comment (tmatsuo) [inline kms/quickstart/phpunit.xml.dist]: small nit: it's 2017

File: kms/quickstart/quickstart.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+$client = new Google_Client();
+
+// Authorize the client using Application Default Credentials
+// @see https://developers.google.com/identity/protocols/application-default-credentials
+$client->useApplicationDefaultCredentials();
+
+// Set the required scopes to access the Key Management Service API
+$client->setScopes(array(
+    'https://www.googleapis.com/auth/cloudkms',
+    'https://www.googleapis.com/auth/cloud-platform'
```
Comment (tmatsuo) [inline kms/quickstart/quickstart.php]: Do you really need both?

Comment (jmdobry) [inline kms/quickstart/quickstart.php]: Nope, remove `https://www.googleapis.com/auth/cloudkms`

File: kms/quickstart/quickstart.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+# [START kms_quickstart]
+# Includes the autoloader for libraries installed with composer
+require __DIR__ . '/vendor/autoload.php';
+
+# Your Google Cloud Platform project ID
+$projectId = 'YOUR_PROJECT_ID';
+
+# The "global" zone will list all keys, can be a specific zone if desired
+$zone = 'global';
```
Comment (jmdobry) [inline kms/quickstart/quickstart.php]: Change variable name throughout from `$zone` to `$location`, per https://github.com/GoogleCloudPlatform/python-docs-samples/pull/754

File: kms/quickstart/quickstart.php [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+ */
+
+# [START kms_quickstart]
+# Includes the autoloader for libraries installed with composer
+require __DIR__ . '/vendor/autoload.php';
+
+# Your Google Cloud Platform project ID
+$projectId = 'YOUR_PROJECT_ID';
+
+# The "global" zone will list all keys, can be a specific zone if desired
```
Comment (jmdobry) [inline kms/quickstart/quickstart.php]: Change comment to `Lists keys in the "global" location.`, per https://github.com/GoogleCloudPlatform/python-docs-samples/pull/754

File: kms/quickstart/quickstart.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+// Set the required scopes to access the Key Management Service API
+$client->setScopes(array(
+    'https://www.googleapis.com/auth/cloud-platform'
+));
+
+// Instantiate the Key Management Service API
+$kms = new Google_Service_CloudKMS($client);
+
+// list all key rings for your project
+$keyRings = $kms->projects_locations_keyRings->listProjectsLocationsKeyRings(
```
Comment (jmdobry) [inline kms/quickstart/quickstart.php]: Last question, does this work? The published discovery doc required that I change the Node.js and Python samples to pass in a full, manually constructed path.

Comment (bshaffer) [inline kms/quickstart/quickstart.php]: Had to add the `$parent` parameter - should work now

**Pull request merged**